hummingbot/cli/hummingbot_application.py

```diff
@@ -226,7 +226,7 @@ def config(self, key: str = None):
 
     async def _unlock_wallet(self):
         choice = await self.app.prompt(prompt="Would you like to unlock your previously saved wallet? (y/n) >>> ")
-        if choice.lower() == "y":
+        if choice.lower() in {"", "y", "yes"}:
```

Comment:
easy for someone to accidentally mash on enter repeatedly and accidentally confirm?

Reply:
fixed